.github/workflows/deploy-schema-worker.yml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019-2024 Tauri Programme within The Commons Conservancy
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-License-Identifier: MIT
+
+name: deploy schema worker
+
+on:
+  push:
+    branches:
+      - dev
+    paths:
+      - '.github/workflows/deploy-schema-worker.yml'
+      - 'crates/tauri-schema-worker/**'
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: cloudflare/wrangler-action@3
+        with:
+          command: deploy
+          workingDirectory: 'packages/schema-worker'
```

Comment:
isn't this in the crates folder?

Reply:
yeah that's correct, I initially wrote the worker in JS but later switched to rust so the directory changed